make fails on macOS  - uchar.h not found

I'm trying to build latest `main` on macOS with

`make PROBE_HOST=hosted HOSTED_BMP_ONLY=0`

and I'm getting:

``` ...
  CC      platforms/hosted/remote/protocol_v3.c
  CC      platforms/hosted/remote/protocol_v3_adiv5.c
  CC      platforms/hosted/bmp_remote.c
  CC      platforms/hosted/bmp_libusb.c
 platforms/hosted/bmp_libusb.c:34:10: fatal error: 'uchar.h' file not found
#include <uchar.h>
         ^~~~~~~~~
1 error generated.
make[1]: *** [bmp_libusb.o] Error 1
make: *** [all] Error 2
$
 ```

--- a/src/platforms/hosted/bmp_libusb.c
+++ b/src/platforms/hosted/bmp_libusb.c
@@ -31,7 +31,13 @@
 #include <libusb.h>
 #include <ftdi.h>
 #endif
+#if (defined __has_include && __has_include(<uchar.h>)) || !defined(__APPLE__)
 #include <uchar.h>
+#else
+//https://sourceware.org/bugzilla/show_bug.cgi?id=17979
+#include <stdint.h>
+typedef int_least16_t char16_t;
+#endif
 #include "cli.h"
 #include "ftdi_bmp.h"
 #include "version.h"

